Stripe/STPAddCardViewController.m

```diff
@@ -162,28 +160,14 @@ - (void)createAndSetupViews {
     self.tableView.dataSource = self;
     self.tableView.delegate = self;
 
-    STPSectionHeaderView *addressHeaderView = [STPSectionHeaderView new];
```

Comment:
Only touched this view controller to pull out some shared fill-address-button logic